tldw_Server_API/app/main.py

```
@@ -678,6 +691,21 @@ def _env_to_bool(v):
+        import sys as _sys
+        _is_pytest = ("pytest" in _sys.modules) or bool(os.getenv("PYTEST_CURRENT_TEST"))
+        if _is_pytest and not _JOBS_TEST_CONTEXT:
+            os.environ.setdefault("CHATBOOKS_CORE_WORKER_ENABLED", "false")
+            os.environ.setdefault("JOBS_METRICS_GAUGES_ENABLED", "false")
+            os.environ.setdefault("JOBS_METRICS_RECONCILE_ENABLE", "false")
+            os.environ.setdefault("AUDIO_JOBS_WORKER_ENABLED", "false")
+            os.environ.setdefault("EMBEDDINGS_REEMBED_WORKER_ENABLED", "false")
+            os.environ.setdefault("JOBS_WEBHOOKS_ENABLED", "false")
+            logger.info("Test context outside Jobs: default-disabling Jobs workers and gauges")
+    except Exception:
+        pass
```

Ruff (0.14.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)